### PR TITLE
Use newer style v resource allocation for Torque (C) and PBS (Python)

### LINKED ISSUE #7355: Torque and PBS driver makes wrongly specified qsub line
With 

`QUEUE_OPTION TORQUE MEMORY_PER_JOB 10gb`

in an ert config, the generated qsub line becomes:

`qsub -rn -NDROGON-159 -q hb120 -l nodes=1:ppn=1:mem=10gb -- /prog/komodo/bleeding-py38-rhel7/root/lib64/pythonsite-packages/equilibrium/bin/komodo_job_dispatch /lustre1/users/havb/havb/01_drogon_design_test/realization-159/iter-0`

which gives an error message: "`failed with exit code 241 and error message: qsub: node(s) specification error`"

This error from qsub goes away if "`:mem=10gb`" is removed from the resource specification.

### PR DESCRIPTION
**Pull request by berland**
**Issue**
Resolves #7355 

**Approach**
_Short description of the approach_

(Screenshot of new behavior in GUI if applicable)

- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [x] Added appropriate release note label
- [x] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).
- [ ] Make sure tests pass locally (after every commit!)

## When applicable
- [ ] **When there are user facing changes**: Updated documentation
- [ ] **New behavior or changes to existing untested code**: Ensured that unit tests are added (See [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)).
- [ ] **Large PR**: Prepare changes in small commits for more convenient review
- [ ] **Bug fix**: Add regression test for the bug
- [ ] **Bug fix**: Create Backport PR to latest release

### CODE REVIEW

**Comment by pinkwah** [comment]
Consider making `_resource_string()` once in driver's `__init__` function, instead of once per realisation. The resource args are equivalent everywhere so we can reduce the lines of code associated with this.

**Comment by berland** [comment]
> Consider making `_resource_string()` once in driver's `__init__` function, instead of once per realisation. The resource args are equivalent everywhere so we can reduce the lines of code associated with this.

fixed!

File: tests/unit_tests/scheduler/test_openpbs_driver.py [hunk 1]
```diff
@@ -93,17 +93,19 @@ def capturing_qsub(monkeypatch, tmp_path):
 
 
 def parse_resource_string(qsub_args: str) -> Dict[str, str]:
-    if " -l " not in qsub_args:
-        return {}
-    args = shlex.split(qsub_args)
-    resource_string = args[args.index("-l") + 1]
+    print(qsub_args)
```
Comment (xjules) [inline tests/unit_tests/scheduler/test_openpbs_driver.py:96]: forgotten print statement?

Comment (berland) [reply tests/unit_tests/scheduler/test_openpbs_driver.py:96]: indeed!

Comment (berland) [reply tests/unit_tests/scheduler/test_openpbs_driver.py]: fixed

**Review by pinkwah — APPROVED** [review]
LGTM

**Pull request merged**